src/OrchardCore/OrchardCore.Email.Abstractions/MailMessage.cs

```diff
@@ -49,47 +48,7 @@ public class MailMessage
         /// <summary>
         /// Gets or sets the message content aka body.
         /// </summary>
-        /// <remarks>This property is work in conjunction with <see cref="IsHtmlBody"/> to determine the body type..</remarks>
-        public string Body { get; set; }
-
-        /// <summary>
-        /// Gets or sets the message content as plain text.
-        /// </summary>
-        [Obsolete("This property is deprecated, please use Body instead.")]
-        public string BodyText
-        {
-            get => Body;
-            set
-            {
-                Body = value;
-                IsHtmlBody = false;
-            }
-        }
-
-        /// <summary>
-        /// Gets or sets whether the message body is an HTML.
-        /// </summary>
-        [Obsolete("This property is deprecated, please use IsHtmlBody instead.")]
-        public bool IsBodyHtml
-        {
-            get => IsHtmlBody;
-            set => IsHtmlBody = value;
-        }
-
-        /// <summary>
-        /// Gets or sets whether the message body is plain text.
-        /// </summary>
-        [Obsolete("This property is deprecated, please use IsHtmlBody instead.")]
-        public bool IsBodyText
-        {
-            get => !IsHtmlBody;
-            set => IsHtmlBody = !value;
-        }
-
-        /// <summary>
-        /// Gets or sets whether the message body is an HTML or not. Default is <c>false</c> which is plain text.
-        /// </summary>
-        public bool IsHtmlBody { get; set; }
```

Comment:
Since this is a published interface, keep the old properties with `Obsolete`, and make them use the new `Body` behind the scenes.